common/config.go

```diff
@@ -98,4 +98,16 @@ const (
 	// Store file location
 	OptStoreFileLocation      = "store-file-path"
 	OptStoreFileLocationAlias = "storefilepath"
+
+	// Private Endpoint
+	OptPrivateEndpoint      = "private-endpoint"
```

Comment:
move to config file

Reply:
not moving, we can add to config

Reply:
If we keep on providing them commandline params, no one will adopt the config file usage. The decision was to stop adding the params to commandline and have those in config instead.

Reply:
I don't remember that decision, we can chat about this